server/conn.go

```diff
@@ -63,16 +62,20 @@ func NewConn(conn net.Conn, user string, password string, h Handler) (*Conn, err
 	}
 	c.closed.Set(false)
 
-	if err := c.handshake(); err != nil {
```

Comment:
can we rever this change?

Reply:
Nevermind, I just added the method to the server packaged and used it in the new proxy package where we create the authed conn. Think this should do what we want!